algorithms_task_2/student/include/impl.h

```diff
@@ -9,4 +10,10 @@
  *
  * @return true if source starts from a prefix
  */
-bool startsWith(const std::string& source, const std::string& prefix);
+bool startsWith(const std::string& source, const std::string& prefix){
+    return source.find(prefix, 0) == 0;
```

Comment:
what if prefix will be at the end? so that will be ineffective

Reply:
The task description says it should start with prefix so I thought I should only check from first position. I didn't realize it could also be a suffix and that I should search the end as well. Could you maybe amend the task description?

Reply:
It should, but it can also be not - that's a software development
I mean that if prefix is not present, this search will be linear, while we can return immediately after comparison of the first chars

Reply:
Oh, I see. Thank you! I'll add a character count as well.